triage/2025/2025-07-22.md

```diff
@@ -0,0 +1,245 @@
+# 2025-07-22 Triage Log
+
+Fairly busy week with improvements outweighing regressions. Most of the regressions were considered acceptable given the circumstances (such as landing a long awaited feature). By far the biggest win comes from being a bit smarter about hashing certain information inside of `DefPathHash`. Since hashing happens quite a lot in th compiler's query system, optimizing when hashing happens can have large performance impacts.
```

Comment:
```suggestion
Fairly busy week with improvements outweighing regressions. Most of the regressions were considered acceptable given the circumstances (such as landing a long awaited feature). By far the biggest win comes from being a bit smarter about hashing certain information inside of `DefPathHash`. Since hashing happens quite a lot in the compiler's query system, optimizing when hashing happens can have large performance impacts.
```